plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -236,15 +237,16 @@ export function handleExecutionResult(
 export function patchCallback(
   instrumentationConfig: PgInstrumentationConfig,
   span: Span,
-  cb: PostgresCallback
+  cb: PostgresCallback,
+  attributes: Attributes
 ): PostgresCallback {
   return function patchedCallback(
     this: PgClientExtended,
     err: Error,
     res: object
   ) {
     if (err) {
-      // span.recordException(err);
+      attributes['error.type'] = err.message;
```

Comment:
Why the `error.type` here is resolved in a different way that the previous cases?

Reply:
oh nice catch! actually the others should be updated to message. Fixed!